instance/prod_config.py.default

```diff
@@ -53,3 +53,6 @@ WHITELISTED_IDS = []
 # Admins will have extra access to create "public" teams, and if ADMINS_ACCESS_ALL_MATCHES
 # is set, they can access admin info for all matches (can pause, cancel, etc.) ANY match.
 ADMIN_IDS = []
+
+
+SPECTATOR_IDS = []
```

Comment:
Can you:
1) Make the newline/spacings consistent here
2) Add a comment above this that describes what it does, and how to use it

Reply:
#The spectator value is for admins, observers, and other personnel (including coaches) that need to be able to join the server, while the game is live or in the warmup. And would otherwise get kicked by the game.

Reply:
> #The spectator value is for admins, observers, and other personnel (including coaches) that need to be able to join the server, while the game is live or in the warmup. And would otherwise get kicked by the game.

i added this to the branch